bucket/pelook.json

```diff
@@ -0,0 +1,20 @@
+{
+    "version": "1.73",
+    "description": "Information tool for Windows EXE, DLL, driver and OBJ files (PE/COFF images)",
+    "homepage": "http://bytepointer.com/tools/index.htm#pelook",
+    "license": {
+        "identifier": "Freeware",
+        "url": "http://bytepointer.com/tools/index.htm"
+    },
+    "url": "http://bytepointer.com/download.php?name=pelook.zip#/dl.zip",
+    "hash": "12baccfb7db419c7533bcf495392e1baf9740c8b3bc558e496201c3c1a046cd7",
+    "bin": "pelook.exe",
+    "checkver": {
+        "url": "http://bytepointer.com/tools/pelook_changelist.htm",
+        "regex": "\\d{4}-\\d{2}-\\d{2}\\s+([\\d.]+)"
+    },
+    "autoupdate": {
+        "url": "http://bytepointer.com/download.php?name=pelook.zip#/dl.zip",
+        "extract_dir": "MagicaVoxel-$version-win64"
```

Comment:
Why is there an extract dir here and not outside?

Reply:
Oops. Forgot to remove that,

Reply:
Ohh. I thought that it needed an extract_dir but this wasn't the correct one!!